libs/market-list/src/lib/market.graphql

```diff
@@ -0,0 +1,53 @@
+fragment SingleMarketFields on Market {
+  id
+  decimalPlaces
+  positionDecimalPlaces
+  state
+  tradingMode
+  fees {
+    factors {
+      makerFee
+      infrastructureFee
+      liquidityFee
+    }
+  }
+  tradableInstrument {
+    instrument {
+      id
+      name
+      code
+      metadata {
+        tags
+      }
+      product {
+        ... on Future {
+          oracleSpecForTradingTermination {
+            id
+          }
+          settlementAsset {
+            id
+            symbol
+            name
+            decimals
+          }
+          quoteName
+        }
+      }
+    }
+  }
+  marketTimestamps {
+    open
+    close
+  }
+  depth {
```

Comment:
market.depth.lastTrade.price should be taken from market depth provider

Reply:
it appears unnecessary in any view so I removed it completely